cloud/tests/conftest.py

```diff
@@ -0,0 +1,91 @@
+import boto3
+import os
+import pytest
+
+
+class CloudTestHelper:
+
+    @staticmethod
+    def prepare_data(*args):
+        s3 = boto3.client('s3')
+        for arg in args:
+            s3.download_file('autogluon-cloud', arg, os.path.basename(arg))
```

Comment:
would be nice to have some documentation here, it isn't obvious how this works without looking at the code that uses it, especially since ordered args are being used instead of keyword args.